gui/wxpython/location_wizard/wizard.py

```diff
@@ -603,9 +614,13 @@ def __init__(self, parent, columns, data=None):
         #
         # add some attributes
         #
-        self.attr1 = wx.ListItemAttr()
+        if globalvar.CheckWxVersion(version=[4, 1, 0]):
+            self.attr1 = wx.ItemAttr()
+            self.attr2 = wx.ItemAttr()
+        else:
+            self.attr1 = wx.ListItemAttr()
+            self.attr2 = wx.ListItemAttr()
         self.attr1.SetBackgroundColour(wx.Colour(238, 238, 238))
```

Comment:
There seems to be other places with (List)ItemAttr, could you put it instead in gui_core/wrap.py to minimize duplication, something like:

```
if globalvar.CheckWxVersion(version=[4, 1, 0]):
    from wx import ItemAttr
else:
    from wx import ListItemAttr as ItemAttr 
```
(might need to use # noqa: F401  for flake checks)

Reply:
Realised it must be preferable to let the system set alternate row colour.
Used `ListCtrl.self.EnableAlternateRowColours()` instead of hard coding with latest commit.